changelogs/current.yaml

```diff
@@ -10,6 +10,12 @@ behavior_changes:
   change: |
     Moved xDS code extensions. If you use the xDS and override extensions_build_config.bzl you will
     need to include the new config_subscriptions explicitly.
+- area: build
```

Comment:
I don't think these need to be in the changelog, as there's no end-user behavior change.

Reply:
@ravenblackx are you OK with me removing them? I think it's good to settle this since I went back and forth.